styles.css

```diff
@@ -0,0 +1,115 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #f4f5f7;
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+}
+
+.header {
+  height: 100px;
+}
+
+.header-container {
+  height: 92px;
+  width: 100%;
+  padding-left: 5%;
+  padding-right: 5%;
+  position: fixed;
+  background-color: white;
+}
+
+.status-bar {
+  height: 44px;
+}
+
+.topbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-top: 10px;
+  list-style: none;
+}
+
+.app-bar {
+  height: 48px;
+  display: flex;
+  justify-content: space-between;
+  padding-top: 12px;
+}
+
+.logo {
+  font-size: 20px;
+  font-weight: 600;
+  color: #6070ff;
+}
+
+.fa-bars {
+  font-size: 1.5em;
+  color: #6070ff;
+}
+
+.nav-items {
+  display: none;
+}
+
+.hero-section {
+  background-image: url("./images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-color: #fff;
+  background-size: cover;
+  background-position: center;
+  padding-top: 70px;
+  height: 718px;
+  border-bottom-left-radius: 120px;
+}
+
+.hero-container {
+  margin: 0 auto;
+  padding: 20px;
+}
+
+h1 {
+  padding-top: 90px;
+  font-size: 46px;
+  margin-top: 12px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+p {
+  margin: 10px 0;
+  line-height: 24px;
+}
+
+.connect {
+  font-size: 20px;
+  font-weight: 40px;
+  padding-top: 12px;
+  text-decoration: none;
+  height: 100px;
+  width: 9px;
+  margin-right: 10px;
+  color: #7f8cff;
+}
```

Comment:
- [ ] As **connect** is an _inline_ element, the property **height** and **width** won't have any effect on it. 
- [ ] **40px** is an invalid value for the property **font-weight**.  For example, if you want to make your text bold,  you may set `font-weight: 700` , and if you want to make your text normal, you may set `font-weight:  400`
You can learn more about [font-weight here](https://www.udacity.com/blog/2021/01/html-css-font-weight.html) 😊 

You can refactor your code this  way:
```
.connect {
  font-size: 20px;
  font-weight: 400;
  padding-top: 12px;
  text-decoration: none;
  margin-right: 10px;
  color: #7f8cff;
}
```